accesshandler/pkg/providers/aws/flask/setup/3.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Finalizing Your Deployment
+configFields:
+---
+
+# Finalizing Your Deployment
+
+You can now test the connection!
+
+Afterwards you will be able to:
+- Create a Flask access rule
+- Create an access request for the Flask deployment
+- Assume into the ECS cluster
+
+To finalise the setup you will need to install [granted-flask](https://pypi.org/project/granted-flask/)
+-  More on this here [docs.commonfate.io/granted-approvals/providers/flask](https://docs.commonfate.io/granted-approvals/providers/flask)
```

Comment:
Move the contents of docs.commonfate.io/granted-approvals/providers/flask into this step rather than linking out to a separate website